helper.go

```diff
@@ -200,7 +216,6 @@ func executeCommand(command string, timeout int, allowFail bool) ExecResult {
 			}
 		} else {
 			er.returnCode = 1
-			er.output = fmt.Sprint(err)
```

Comment:
Why don't you want no error output if there was an error?

Reply:
The error and the program's output are two different things. We don't want to overwrite the actual output if it was there and that's why another field, `err` has been introduced to `ExecResult`.